core/src/main/java/com/exadel/core/schedulers/UpdateContentSchedulerTask.java

```diff
@@ -59,8 +61,7 @@ private void deactivate(Config config) {
     public void run() {
         try {
             LOGGER.info("=== UpdateContentSchedulerTask is now running");
-            beautyShopService.updateBrands();
-            beautyShopService.updateTags();
+            Set<String> brands = beautyShopService.updateBrands();
             beautyShopService.updateProducts("maybelline");
             LOGGER.info("=== UpdateContentSchedulerTask is finished");
```

Comment:
Please remove === sign as usless

Reply:
Thanks, just used it in debugging and forgot to remove it.